paasta_tools/kubernetes_tools.py

```diff
@@ -486,12 +495,40 @@ def get_autoscaling_params(self) -> AutoscalingParamsDict:
             defaults=default_params,
         )
 
+    # TODO: move the default scaling policy to system paasta configs
+    # TODO: allow per-instance overrides in soa-configs, and merge this function
```

Comment:
I'm not sure if we necessarily need per-instance overrides, so let's not add it as a TODO?